runtimes/ruby/ruby-on-rails.md

```diff
@@ -68,6 +68,14 @@ releases, but, hey, we're human!
 Due to current landscape in ruby applications, the default version is the
 greatest 2.3.Y. We provide also the 2.2.Y and 2.1.Y versions.
 
+## Choose rackup application server
+
+By default your application will be run using uwsgi. If you prefer, you can select another server
```

Comment:
can you add a link to the uwsgi website, and/or a small paragraph? uWSGI is not really known in the ruby community. Maybe write a small comparison between the two (eg uWSGI is faster, but puma has better support, something like that)